src/vs/workbench/contrib/notebook/common/services/notebookSimpleWorker.ts

```diff
@@ -64,15 +64,13 @@ class MirrorCell {
 			}
 		}
 
-		// note: hash has not updated within the Promise.all since we must retain order
-		const digests = await Promise.all(this.outputs.flatMap(op =>
-			op.outputs.map(o => crypto.subtle.digest('sha-1', o.data.buffer))
-		));
+		const digests = this.outputs.flatMap(op =>
+			op.outputs.map(o => hash(Array.from(o.data.buffer)))
```

Comment:
Using the same technique used in renderer process to compute of hashes of outputs.
Also makes it simpler
A separate PR would be to change `computeDiff` to make it sync, however thats a bigger change and leaving that for later.
@rebornix /cc